src/components/Dashboard/FilingHistoryList/FilingTemplate.vue

```diff
@@ -53,7 +53,10 @@
       </div>
     </v-expansion-panel-header>
 
-    <v-expansion-panel-content class="pt-5">
+    <v-expansion-panel-content
+      eager
+      class="pt-5"
+    >
```

Comment:
Based on discussion thread below added `eager` to the `v-expansion-panel-content`. This way panel content is generated and can be tested in the unit test.
https://github.com/vuejs/vue-test-utils/issues/1827

Tested a bit to make sure functionality and user experience is same.

Reply:
This is very useful! I know some previous unit tests were either disabled or not completed because expansion panels did not seem to open in the unit tests. Thanks for the research and sharing 👍 

PS Consider sharing this at the next Developer Demo.